src/core/server/base.js

```diff
@@ -83,6 +83,11 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
     app.get('/', (req, res) => res.redirect(302, '/search'));
   }
 
+  if (appInstanceName === 'disco' && isDevelopment) {
+    app.get('/', (req, res) =>
```

Comment:
react-router has a [`<Redirect />`](https://github.com/reactjs/react-router/blob/master/docs/API.md#redirect) component, can you add that to the router in disco/routes.js when in development instead?